modules/cli/cmd/devrunner/setconfig.go

```diff
@@ -0,0 +1,287 @@
+package devrunner
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/astriaorg/astria-cli-go/modules/cli/cmd"
+	"github.com/astriaorg/astria-cli-go/modules/cli/cmd/devrunner/config"
+	util "github.com/astriaorg/astria-cli-go/modules/cli/cmd/devrunner/utilities"
+	"github.com/pelletier/go-toml/v2"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// setConfigCmd represents the set-config root command
+var setConfigCmd = &cobra.Command{
+	Use:   "set-config",
```

Comment:
This usage of dashes isn't consistent with the rest of our cli's api, e.g. `createaccount`, `ibctransfer`.